cypress/features/regression/sidebar.feature

```diff
@@ -1,29 +1,15 @@
 Feature: Sidebar component
   I want to test Sidebar component
 
-  @positive
-  Scenario: CloseIcon has the border outline
-    When I open default "Sidebar Test" component in noIFrame with "sidebar" json from "commonComponents" using "default" object name
-    Then closeIcon has the border outline color "rgb(255, 181, 0)" and width "3px"
-
   @positive
   Scenario: Enable enableBackgroundUI checkbox for a Sidebar component
     When I open default "Sidebar Test" component in noIFrame with "sidebar" json from "commonComponents" using "enableBackgroundUI" object name
     Then Sidebar component has enabled background UI
 
   @positive
-  Scenario: Disable enableBackgroundUI checkbox for a Sidebar component
-    When I open default "Sidebar Test" component in noIFrame with "sidebar" json from "commonComponents" using "enableBackgroundUIFalse" object name
-    Then Sidebar component has disabled background UI
-
-  @positive
-  Scenario Outline: Sidebar Pod position to <position>
-    When I open default "Sidebar Test" component in noIFrame with "sidebar" json from "commonComponents" using "<nameOfObject>" object name
-    Then Sidebar position value is set to "<position>"
-    Examples:
-      | position | nameOfObject  |
-      | left     | positionLeft  |
-      | right    | positionRight |
+  Scenario: Sidebar Pod position to left
```

Comment:
isn't it possible to cover it with `chromatic` ?

Reply:
we don't have a story with left position